### PR TITLE
Add /lookup specification

### LINKED ISSUE #4: Add `/lookup` endpoint specification
**Describe the solution you'd like**
The [official documentation](https://nominatim.org/release-docs/develop/api/Lookup/) describes the `/lookup` endpoint. We should add it to `paths` section and describe all accepted parameters. The response schema (for the `geocodejson` format, see #2) will be the same of `/search` endpoint.

**Additional context**
Some useful resources:
- [nominatim lookup endpoint](https://nominatim.org/release-docs/develop/api/Lookup/)
- [paths object in openapi](https://spec.openapis.org/oas/v3.1.0#paths-object)

### PR DESCRIPTION
**Pull request by tegola**
Closes #4

### CODE REVIEW

**Comment by jenkin** [comment]
After #8 merging, please rebase main branch into this one and re-run `make checksum` to fix checksum file. Thanks!

**Comment by tegola** [comment]
Rebased, no need to run `make checksum` (I did anyway to check there weren't changes) since I had already used `shasum` to generate the checksum.

**Comment by tegola** [comment]
Uh, my bad. There is actualy a change to push. The slow vscode refresh and a misunderstanding made me think there wasn't any. Commit incoming.

File: docs/nominatim.openapi.json [hunk 1]
```diff
@@ -299,6 +299,171 @@
                     "Search"
                 ]
             }
+        },
+        "/lookup": {
+            "get": {
+                "summary": "Query the address and other details of one or multiple OSM objects like node, way or relation.",
+                "operationId": "123",
```
Comment (jenkin) [inline docs/nominatim.openapi.json:306]: Should be `lookup`, not `123`...

Comment (jenkin) [inline docs/nominatim.openapi.json]:
```suggestion
                "operationId": "lookup",
```

File: docs/nominatim.openapi.json [hunk 2]
```diff
@@ -299,6 +299,171 @@
...
+                "description": "Returns the details of an OSM object.",
+                "parameters": [
+                    {
```
Comment (jenkin) [inline docs/nominatim.openapi.json:309]: You totally missed the only required parameter `osm_ids` (see the [official docs](https://nominatim.org/release-docs/develop/api/Lookup/#endpoint))... 😄 

```suggestion
                "parameters": [
                    {
                        "schema": {
                            "type": "string",
                            "pattern": "^([RWN][0-9]+,?)+$"
                        },
                        "in": "query",
                        "name": "osm_ids",
                        "required": true,
                        "description": "Comma-separated list of OSM ids each prefixed with its type, one of node(N), way(W) or relation(R). Up to 50 ids can be queried at the same time."
                    },
                    {
```

File: docs/nominatim.openapi.json [hunk 3]
```diff
@@ -299,6 +299,171 @@
...
+                        "name": "accept-language",
+                        "description": "Preferred language order for showing search results, overrides the value specified in the [Accept-Language HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language). Either use a standard RFC2616 accept-language string or a simple comma-separated list of language codes."
+                    },
+                    {
+                        "schema": {
+                            "type": "string"
+                        },
+                        "in": "query",
+                        "name": "countrycodes",
+                        "description": "Limit search results to one or more countries. `<countrycode>` must be the [ISO 3166-1alpha2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) code. Each place in Nominatim is assigned to one country code based on OSM country boundaries. In rare cases a place may not be in any country at all, for example, in international waters."
```
Comment (jenkin) [inline docs/nominatim.openapi.json]: There is no mention to `countrycodes` in the [official docs](https://nominatim.org/release-docs/develop/api/Lookup/#endpoint)...

```suggestion
```

**Review by jenkin — CHANGES_REQUESTED** [review]
Very nice, thank you! Just three more changes and this PR can be merged... 😃

**Comment by jenkin** [comment]
@tegola Please rebase the main branch another time, I fixed the git hooks issue.

**Review by jenkin — APPROVED** [review]

**Pull request merged**